content/en/docs/concepts/storage/volumes.md

```diff
@@ -1086,7 +1086,7 @@ persistent volume:
 * If you are running Kubernetes Version 1.25 or 1.26, you must enable
   the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
   named `CSINodeExpandSecret` for each kube-apiserver and for the kubelet on every
-  node. In Kubernetes version 1.27 this feature has been enabled by default
+  node. In Kubernetes version 1.29 this feature has been enabled by default
```

Comment:
This statement about history is correct and should not be changed.
If you want, you can add that this feature is GA'ed in v1.29.

Reply:
Better I kept this unchanged.